lit_parrot/speed_monitor.py

```diff
@@ -89,8 +102,15 @@ def get_flops_available(device: torch.device, precision: str) -> Optional[float]
                     "MFU cannot be calculated and reported."
                 )
     elif device.type == "xla":
-        # TODO
-        ...
+        from torch_xla.experimental import tpu
+        try:
+            tpu_generation = tpu.get_tpu_env()['TYPE']
+            return TPU_AVAILABLE_FLOPS[tpu_generation]
```

Comment:
Use xenv as in https://github.com/Lightning-AI/lit-gpt/pull/172/files#diff-d83a788c6cc66e57022968bc902a9c15ff5e29f19e55f0f0aaadaae37f132cb8R98-R100?